### PR TITLE
Merge latest i18n updates

### PR DESCRIPTION
**Pull request by twelch**
This pull request does a few things, please test it out (try translation:sync and storybook) and let's confirm together it's all working:
- moves data/scripts out to the top-level
- adds scripts/extractExtraTerms.ts which extracts translatable strings from objectives.json and metrics.json. this scripts now gets run during `translation:extract`
- updates SizeCard to translate class names displayed used in tables
- updates sketch example to include full translation metadata for user attributes (Island, Designation)

### CODE REVIEW

File: examples/sketches/nearshore-test.geojson.json [hunk 1]
```diff
@@ -78,9 +84,10 @@
                 "valueLabel": [
                     "Santa Maria"
                 ],
+                "formElementId": 2990,
                 "alternateLanguages": {
                     "pt": {
-                        "label": null,
+                        "label": "Ilha",
```
Comment (twelch) [inline examples/sketches/nearshore-test.geojson.json:90]: I updated the sketch to have the latest attribute metadata including translations.  This was probably an old sketch example before we setup the translations

File: examples/sketches/nearshore-test.geojson.json [hunk 2]
```diff
@@ -66,7 +66,13 @@
...
                 "fieldType": "ComboBox",
                 "valueLabel": "Fully Protected",
-                "alternateLanguages": {}
+                "formElementId": 2987,
+                "alternateLanguages": {
+                    "pt": {
+                        "label": "Designação",
+                        "valueLabel": "Totalmente Protegido"
+                    }
+                }
```
Comment (twelch) [inline examples/sketches/nearshore-test.geojson.json:75]: I updated the sketch to have the latest attribute metadata including translations.  This was probably an old sketch example before we setup the translations

File: package.json [hunk 3]
```diff
@@ -18,9 +18,9 @@
...
     "publish:data": "geoprocessing publish:data",
-    "install:data": "mkdir -p data/bin && cp -r node_modules/@seasketch/geoprocessing/dist/data/bin/* data/bin && mkdir -p data/scripts && cp -r node_modules/@seasketch/geoprocessing/dist/data/scripts/* data/scripts",
+    "install:scripts": "mkdir -p data/bin && cp -r node_modules/@seasketch/geoprocessing/dist/data/bin/* data/bin && mkdir -p scripts && cp -r node_modules/@seasketch/geoprocessing/dist/base-project/scripts/* scripts",
     "translation:install": "rm -rf src/i18n/baseLang && cp -r node_modules/@seasketch/geoprocessing/dist/base-project/src/i18n/baseLang src/i18n",
-    "translation:extract": "rm -rf src/i18n/lang/en && NODE_ENV=development npx babel 'src/**/*.{ts,tsx}' > /dev/null && mv extractedTranslations/en src/i18n/lang/en && rm -rf extractedTranslations && echo 'Translations extracted to src/lang/en'",
+    "translation:extract": "rm -rf src/i18n/lang/en && NODE_ENV=development npx babel 'src/**/*.{ts,tsx}' > /dev/null && mv extractedTranslations/en src/i18n/lang/en && rm -rf extractedTranslations && npx ts-node scripts/extractExtraTerms.ts && echo 'Translations extracted to src/lang/en'",
```
Comment (twelch) [inline package.json:23]: run our new script for extracting extra terms

File: package.json [hunk 4]
```diff
@@ -18,9 +18,9 @@
     "reimport:data": "start-server-and-test 'http-server data/dist -c-1 -p 8001' http://localhost:8001 reimport:data_",
     "reimport:data_": "geoprocessing reimport:data",
     "publish:data": "geoprocessing publish:data",
-    "install:data": "mkdir -p data/bin && cp -r node_modules/@seasketch/geoprocessing/dist/data/bin/* data/bin && mkdir -p data/scripts && cp -r node_modules/@seasketch/geoprocessing/dist/data/scripts/* data/scripts",
+    "install:scripts": "mkdir -p data/bin && cp -r node_modules/@seasketch/geoprocessing/dist/data/bin/* data/bin && mkdir -p scripts && cp -r node_modules/@seasketch/geoprocessing/dist/base-project/scripts/* scripts",
```
Comment (twelch) [inline package.json:21]: switch to installing scripts into the top-level of our project, instead of into data

File: package.json (hunk 3 above)
Comment (avmey) [reply package.json:23]: Due to the echo I was checking to see if the extra terms were in src/i18n/lang/en. I would update the echo to `Translations extracted to src/i18n/lang/en and src/i18n/extraTerms.json`. Other than that it worked great.

**Pull request merged**